pkg/etcd/client.go

```diff
@@ -145,12 +159,31 @@ func isRetryableError(rpcName string) retry.IsRetryable {
 			return false
 		}
 		if rpcName == EtcdRevoke {
-			if etcdErr, ok := err.(rpctypes.EtcdError); ok && etcdErr.Code() == codes.NotFound {
+			if etcdErr, ok := err.(v3rpc.EtcdError); ok && etcdErr.Code() == codes.NotFound {
 				// it means the etcd lease is already expired or revoked
 				return false
 			}
 		}
 
```

Comment:
ref https://github.com/pingcap/tiflow/blob/master/dm/pkg/etcdutil/etcdutil.go#L51-L66

Reply:
We can extract a function, such as `EtcdRetryableError() bool` in the future, similar to the work in https://github.com/pingcap/tiflow/issues/3242

Reply:
Done, PTAL